src/tasks/Tasks.ts

```diff
@@ -655,14 +658,14 @@ export abstract class ATouringTask implements ITouringTask {
             for (const attr of cellData.highlightData.filter((data) => data.category !== undefined)) {
               const indices = this.ranking.getAttributeDataDisplayed(attr.column).reduce((indices, cat, index) => cat === attr.category ? [...indices, index] : indices, []);
               for (const index of indices) {
-                const elem = d3.select(`.lineup-engine main .lu-row[data-index="${index}"][data-agg="detail"] [data-id="${id}"]`);
+                const elem = d3.select(focusedLineupNode).select(` .lineup-engine main .lu-row[data-index="${index}"][data-agg="detail"] [data-id="${id}"]`);
                 if (!elem.empty()) {
                   const setDarker = elem.classed(`${cssClass}-1`); //if previous class is already set
                   elem.classed(`${cssClass}-${i}`, true)
                     .classed(`${cssClass}-dark`, setDarker);
 
-                  const catId = d3.select(`.lineup-engine header .lu-header[title^="${attr.label}"]`).attr('data-col-id');
-                  d3.select(`.lineup-engine main .lu-row[data-index="${index}"] [data-id="${catId}"]`).classed(`${cssClass}-border`, true);
+                  const catId = d3.select(` .lineup-engine header .lu-header[title^="${attr.label}"]`).attr('data-col-id');
```

Comment:
Shouldn't this also use the `focusedLineupNode`?

```suggestion
                  const catId = focusedLineupNode.select(`.lineup-engine header .lu-header[title^="${attr.label}"]`).attr('data-col-id');
```

Reply:
Yes.